.gitignore

```diff
@@ -31,6 +31,8 @@ objsamples/ET_Client.py
 objsamples/config.python
 objsamples/*.pyc
 config.python
+FuelSDK/Public_WebAppTests/config.python
```

Comment:
Please add template file for the config needed to run the tests in this folder.

Reply:
Template added.